book/website/reproducible-research/licensing/licensing-software.md

```diff
@@ -31,82 +46,91 @@ To make some sense of this variety, we can categorize them as follows.
-            <td>Research-only No&nbsp;copying No&nbsp;modification</td>
+        <td>BSD<sup>5</sup> MIT<sup>6</sup> Apache</td>
+            <td>Research Only: No&nbsp;copying, No&nbsp;modification</td>
         </tr>
     </tbody>
+    <caption>
+      <div class="footnote">
+        <sup>1</sup>GPL: GNU General Public License <sup>2</sup>CDDL: Common Development and Distribution License <sup>3</sup>LGPL: GNU Lesser General Public License <sup>4</sup> MPL: Mozilla Public License <sup>5</sup> BSD: Berkley Software Distribution <sup>6</sup> MIT: Massachusetts Instititue of Technology
+      </div>
+      Licenses can either be Free or Propietary, with Free Licenses further classified as Copyleft or Permissive.
+    </caption>
 </table>
 
-<sup>1</sup>GNU General Public License
-<sup>2</sup>CDDL Common Development and Distribution License
-<sup>3</sup>GNU Lesser General Public License
-<sup>4</sup>MPL Mozilla Public License
-<sup>5</sup>BSD MIT Berkeley Software Distribution Massachusetts Institute
-
-## Free software
+(rr-licensing-software-free)=
+## Free Software
 
-The main distinction among software licenses is between Free software and proprietary software.
-Free software come with license terms that give you ([as defined by GNU](https://www.gnu.org/philosophy/free-sw.html))
+Software licenses are either free or proprietary. 
+Free software comes with license terms that give you ([as defined by GNU](https://www.gnu.org/philosophy/free-sw.html)):
 
-* freedom 0: The freedom to run the program as you wish, for any purpose.
-* freedom 1: The freedom to study how the program works, and change it so it does your computing as you wish.
+* _Freedom 0_: The freedom to run the program as you wish, for any purpose.
+* _Freedom 1_: The freedom to study how the program works, and change it, so it does your computing as you wish.
 Access to the source code is a precondition for this.
-* freedom 2: The freedom to redistribute copies so you can help others in your community.
-* freedom 3: The freedom to distribute copies of your modified versions to others.
-By doing this you can give the whole community a chance to benefit from your changes. Access to the source code is a precondition for this.
+* _Freedom 2_: The freedom to redistribute copies so you can help others in your community.
+* _Freedom 3_: The freedom to distribute copies of your modified versions to others.
+By doing this, you can give the whole community a chance to benefit from your changes. Access to the source code is a precondition for this.
 
-These four freedoms together effectively neutralize copyright: **freedoms 1 and 3** let you create derivative works, and **freedoms 2 and 3** let you make copies.
+These four freedoms together effectively neutralize copyright; *Freedoms 1* and *3* let you use the original software and share it with others. 
+*Freedoms 2* and *3* let you create derivative works based on the software and share these with others.
```

Comment:
@paulowoicho I think my proposed changes got garbled in the translation from the 1-4 numbering back to the 0-3 numbering. I've updated them below.
```suggestion
These four freedoms together effectively neutralize copyright; *Freedoms 0* and *2* let you use the original software and share it with others. 
*Freedoms 1* and *3* let you create derivative works based on the software and share these with others.
```

Reply:
Change applied in commit a718e78

Reply:
Ahh okay. thanks for that!